README.md

```diff
@@ -16,6 +16,14 @@ For example if there is an image subfolder under your extension project workspac
 
 If you have any requirements or dependencies, add a section describing those and how to install and configure them.
 
+## Contribution
+
```

Comment:
```suggestion
How to develop this extension
```
So GitHub does have a suggestion feature, though it looks like comments can only be on a line, not on a highlighted section.